### PR TITLE
Add Ansible for audit_rules_system_shutdown

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Fix mistake in Bash remediation for `audit_rules_system_shutdown`
- Add Ansible remediation for `audit_rules_system_shutdown` (inspired by remediation for `audit_rules_immutable`)
- Clarify task name in Ansible for `audit_rules_immutable`

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule audit_rules_system_shutdown:
 Found change in bash remediation.
 Ansible remediation newly added.

**Recommended tests to execute:**
 build_product ol7
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_system_shutdown
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-ol7-ds.xml audit_rules_system_shutdown

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_system_shutdown/bash/shared.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # files to check if '-f .*' setting is present in that '*.rules' file already.
 # If found, delete such occurrence since auditctl(8) manual page instructs the
```
[the remark this answers is not in the document]
Comment (yuumasato) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_system_shutdown/bash/shared.sh:9]: @vojtapolasek So does it mean `audit_rules_immutable` should run after `audit_rules_system_shutdown`?

**Comment by vojtapolasek** [comment]
Rules can be remediateed in any order. The only requirement is that the -e option is the last one in /etc/audit/audit.rules.

**Comment by yuumasato** [comment]
> Rules can be remediateed in any order. The only requirement is that the -e option is the last one in /etc/audit/audit.rules.

If they can be remediated in any order, I don't understand how `-e` option will be the last in `/etc/audit/audit.rules`.
I guess `augenrules` can handle that, but how about `auditctl`?

**Comment by mildas** [comment]
**Changes identified:**
Rule audit_rules_system_shutdown:
 Found change in bash remediation.
 Ansible remediation newly added.

**Recommended tests to execute:**
 build_product ol8
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol8-ds.xml audit_rules_system_shutdown
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-ol8-ds.xml audit_rules_system_shutdown

**Comment by vojtapolasek** [comment]
Looks good, thank you.

**Pull request merged**